charts/exposecontroller/templates/role.yaml

```diff
@@ -36,3 +36,18 @@ rules:
   - watch
   - patch
   - update
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ .Chart.Name }}
+  # This only grants permissions within the "default" namespace.
+  namespace: default
```

Comment:
Can we remove specifying the default namespace?  Folks will install different things in different namespaces so best to avoid using a namespace here.

Reply:
@rawlingsj  exposecontroller needs permissions to list and get services in the default namespace specifically, at least that's what I observed from the logs

```
W0404 07:28:37.387957       1 controller.go:259] Could not find kubernetes service in the default namespace so we could not detect whether to use http or https as the apiserver protocol. Error: services "kubernetes" is forbidden: User "system:serviceaccount:test2:exposecontroller" cannot get services in the namespace "default"
```

Reply:
@jstrachan @rawlingsj I added the "default" namespace role because of this error, the role fore `Release.Namespace` already existed. 

According to the error, it needs to list and get services in the default namespace. 
Now is that necessary for exposecontroller? as the log says